xen/arch/arm/domain_build.c

```diff
@@ -1332,7 +1332,8 @@ static int handle_node(struct domain *d, struct kernel_info *kinfo,
         return make_timer_node(d, kinfo->fdt, node);
 
 #ifdef CONFIG_HAS_COPROC
-    if ( device_get_class(node) == DEVICE_COPROC )
+    if ( (device_get_class(node) == DEVICE_COPROC) &&
```

Comment:
Is dt_device_used_by() the only way to realize that the coproc is not going to be shared?
What about xen,coproc property?